### ISSUE TITLE
[RFC] add ocabot config file per repo options

### ISSUE THREAD

**Issue by legalsylvain** [OCA/oca-github-bot#108]
Do you think it could be relevant to add the possibility to have custom config file in some repo, to overload some values present in the environment file. (For exemple for the value that mentions the number of APPROVALS required) that can depends on each repo policy. For exemple OpenUpgrade project has a more optimistic merging policy. (One approval required)

**Comment by sbidoul** [comment]
Could make sense, yes. That would require a significant refactoring of the configuration handling code if we want to keep it clean and manageable.

**Comment by legalsylvain** [comment]
> That would require a significant refactoring of the configuration handling code if we want to keep it clean and manageable.

yes, it'll take some thought.

I'll not work immediately on that feature, but we can let it opened. I think that if oca-github-bot comes with more non-consensual features like rebase command (https://github.com/OCA/oca-github-bot/pull/71#pullrequestreview-283718787), maybe it will be usefull to enable / disable for some repository there features via ``BOT_TASKS`` and ``BOT_TASKS_DISABLED``, depending of the choice of the maintainer(s).

**Comment by sbidoul** [comment]
Agreed. Some `.oca-github-bot.yml` file at the repo root could override the global config options.
